sdks/java/extensions/google-cloud-platform-core/src/test/java/org/apache/beam/sdk/extensions/gcp/storage/GcsResourceIdTest.java

```diff
@@ -168,7 +169,7 @@ public void testGetFilename() throws Exception {
 
   @Test
   public void testResourceIdTester() throws Exception {
-    FileSystems.setDefaultPipelineOptions(TestPipeline.testingPipelineOptions());
+    FileSystems.setDefaultPipelineOptions(PipelineOptionsFactory.as(GcpOptions.class));
```

Comment:
LGTM
Made this just PipelineOptionsFactory.create() on merge since the as(GcpOptions.class) does nothing